foundation-extensions/profile/profile-api-references.md

```diff
@@ -501,6 +663,14 @@ UserProfile.removeUserAttributes(Arrays.asList("username", "usertype"));
 
 ### Objective-C
 
+**Syntax**
+
+```objectivec
++ (void) removeUserAttributesWithAttributeNames:(NSArray<NSString *> * _Nonnull)
```

Comment:
this syntax does not seem to match the example below, which one is correct?

Reply:
The API name should be `removeUserAttributesWithAttributeNames`, I will update the example code. 
https://github.com/adobe/aepsdk-compatibility-ios/blob/12c5e8681905140f0a09e20e2339fed85f4d9ea8/ACPUserProfile/ACPUserProfile.m#L36

Reply:
:(